src/backend.md

```diff
@@ -630,6 +630,55 @@ For serverless projects - method 3 is preferred, but not always. When you have a
       - updateBankAccount.controller.test.ts
 ```
 
+1. One test should include one action as a call of controller or job.
+
+```typescript
+   // bad
+
+   it((caseX) => {
+     // ...
+    await postSqsJob(JobName.name, {...});
+    // ...
+    await postSqsJob(JobName.name, {...});
```

Comment:
It is a good proposal but in some rare cases we need to check something on second, repeat call. For example that we didn't not create something twice. What should we do in these cases?

Reply:
IMO preset quality is the key to a cool and simple test.
Preset is a state of the system. Every time we can configure this state.
I saw several time tests with double acts. It was always the engineer’s choice to make two calls instead of creating 2 tests with form preset.

Reply:
>For example that we didn't not create something twice

I mean, I didn't see something that we can't configure. We can set up all that we need: DB row, redis, request headers and data.
We can mock something as an ultimate solution.